pkg/scheduler/factory/plugins.go

```diff
@@ -409,6 +409,7 @@ func RegisterCustomPriorityFunction(policy schedulerapi.PriorityPolicy) string {
 
 	if pcf == nil {
 		klog.Fatalf("Invalid configuration: Priority type not found for %s", policy.Name)
```

Comment:
Thanks @tedyu 
Doesn't `klog.Fatalf` cause the program exit?
Ref: https://github.com/kubernetes/klog/blob/master/klog.go#L1341

Reply:
I see logging.printf() call.
Do you see the exit() call ?

Reply:
So this `return ""` will never be called, right? Or something I missed.

Reply:
I reopened to trigger test run for pull-kubernetes-node-e2e (which always fails)